pkg/sm/config.go

```diff
@@ -32,20 +32,21 @@ func NewConfig(env env.Environment) (*Config, error) {
 	return config.Sm, nil
 }
 
-// Config type holds SM Client config properties
-type Config struct {
+// Settings type holds SM Client config properties
+type Settings struct {
 	User              string
 	Password          string
 	Host              string
-	OsbAPI            string
-	RequestTimeout    time.Duration
-	SkipSslValidation bool
+	OsbAPI            string        `mapstructure:"osb_api"`
```

Comment:
`OSBAPI`, see https://github.com/golang/go/wiki/CodeReviewComments#initialisms
What is this URL, path, version? Make it clear in the name.